notebooks/Rural_Access_Index.ipynb

```diff
@@ -0,0 +1,359 @@
+     "shell.execute_reply": "2022-05-24T22:03:34.504387Z",
+     "shell.execute_reply.started": "2022-05-24T22:03:34.500962Z"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stdout",
+     "output_type": "stream",
+     "text": [
+      "/home/jovyan/workspace/road_analytics/Armenia\n"
+     ]
+    }
+   ],
+   "source": [
+    "folder = f'/home/jovyan/workspace/road_analytics/{model_place}'\n",
+    "\n",
+    "if isdir(folder):\n",
+    "    print(folder)"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 4,
+   "id": "1a923ae9-8f01-46af-993c-2ba7e363a44e",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-05-24T22:03:35.080477Z",
+     "iopub.status.busy": "2022-05-24T22:03:35.080099Z",
+     "iopub.status.idle": "2022-05-24T22:03:35.360498Z",
+     "shell.execute_reply": "2022-05-24T22:03:35.359953Z",
+     "shell.execute_reply.started": "2022-05-24T22:03:35.080452Z"
+    },
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "project = Project()\n",
+    "project.open(folder)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "c0ed9c6a-dc9c-41fd-8570-7c0a3f3755a5",
+   "metadata": {},
+   "source": [
+    "### Find rural areas in Armenia"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 5,
+   "id": "fb899c50-d16a-4287-a303-a74e12be4df6",
+   "metadata": {
+    "execution": {
+     "iopub.execute_input": "2022-05-24T22:03:36.227089Z",
+     "iopub.status.busy": "2022-05-24T22:03:36.226712Z",
+     "iopub.status.idle": "2022-05-24T22:03:39.452150Z",
+     "shell.execute_reply": "2022-05-24T22:03:39.451468Z",
+     "shell.execute_reply.started": "2022-05-24T22:03:36.227065Z"
+    },
+    "tags": []
+   },
+   "outputs": [
+    {
+     "name": "stderr",
+     "output_type": "stream",
+     "text": [
+      "/srv/conda/envs/saturn/lib/python3.9/site-packages/pandas/core/frame.py:684: ShapelyDeprecationWarning: __len__ for multi-part geometries is deprecated and will be removed in Shapely 2.0. Check the length of the `geoms` property instead to get the  number of parts of a multi-part geometry.\n",
+      "  data = list(data)\n",
+      "/srv/conda/envs/saturn/lib/python3.9/site-packages/pandas/core/frame.py:684: ShapelyDeprecationWarning: Iteration over multi-part geometries is deprecated and will be removed in Shapely 2.0. Use the `geoms` property to access the constituent parts of a multi-part geometry.\n",
+      "  data = list(data)\n",
+      "/srv/conda/envs/saturn/lib/python3.9/site-packages/geopandas/geodataframe.py:2196: UserWarning: CRS mismatch between the CRS of left geometries and the CRS of right geometries.\n",
+      "Use `to_crs()` to reproject one of the input geometries to match the CRS of the other.\n",
+      "\n",
+      "Left CRS: None\n",
+      "Right CRS: EPSG:4326\n",
+      "\n",
+      "  return geopandas.overlay(\n"
+     ]
+    }
+   ],
+   "source": [
+    "rural_areas = select_rural_areas(model_place)"
```

Comment:
These function calls mean very little to the user (the steps mean nothing), so it is a little silly to divide into sections.

It is preferrable to call the computation at a higher level of detail (say each population point you say if it is within the distance threshold or not) and then aggregate by different levels, which is something that would allow somebody to use it to do the analysis at the level they would like (from hexbin to whole country).